CHANGELOG.md

```diff
@@ -115,6 +125,10 @@ behavior
 - Fixed bug where `cmr_custom_host` variable was not properly forwarded into `archive`, `ingest`, and `sqs-message-remover` modules from `cumulus` module
 - Fixed bug where `parse-pdr` set a granule's provider to the entire provider record when a `NODE_NAME`
   is present. Expected behavior consistent with other tasks is to set the provider name in that field.
+- **CUMULUS-2548**
+  - Fixed `@cumulus/ingest/HttpProviderClient.sync` to
```

Comment:
There should probably be a 'changed' note about the new redirect schema/config updates.

Reply:
Is this sufficient?

https://github.com/nasa/cumulus/pull/2356/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR19-R20